CHANGELOG.md

```diff
@@ -14,6 +14,11 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Configuration setting for the websocket URL that the ASB connects to in order to receive price ticker updates.
   Can be configured manually by editing the config.toml file directly.
   It is expected that the server behind the url follows the same protocol as the [Kraken websocket api](https://docs.kraken.com/websockets/).
+- Registration and discovery of ASBs using the [libp2p rendezvous protocol](https://github.com/libp2p/specs/blob/master/rendezvous/README.md).
+  ASBs can register with a rendezvous node upon startup and, once registered, can be automatically discovered by the CLI using the `list-sellers` command.
+  The rendezvous node address (`rendezvous_point`), as well as the ASB's external addresses (`external_addresses`) to be registered, is to be configured in the `network` section of the ASB config file.
```

Comment:
nitpick: "is ~~to be~~ configured in the `network` section of the ASB config file."